tools/flash-shelly.py

```diff
@@ -163,7 +163,7 @@ def __init__(self, host=None, wifi_ip=None, fw_type=None, device_url=None, info=
     self.flash_label = "Latest:"
     self.force_flash = False
 
-  def is_host_reachable(self, host):
+  def is_host_reachable(self, host, isflashing=False):
```

Comment:
isflashing -> is_flashing, ilikemywordsdelimitedpleasethankyouverymuch :)

Reply:
okdone, is it better now 😄